domain/entity/shift_request.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+from sqlalchemy import Column, String, DateTime, ForeignKey, Integer
+from sqlalchemy.orm import relationship
+
+from domain.base import Base
+
+
+class ShiftRequest(Base):
+    __tablename__ = 'shift_request'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    user_id = Column(Integer, ForeignKey('user.id'), name='user_id', nullable=False)
+    title = Column(String(29), name='title', nullable=False)
+    startTime = Column(DateTime, name='from', nullable=False)
+    endTime = Column(DateTime, name='to', nullable=False)
+    status = Column(String(12), name='status', default='waiting', nullable=False)
```

Comment:
Please use an enum for this. It means that we can't accidentally use incorrect data and enumerates all valid options for future maintainers.

Reply:
This StackOverflow post can get you started -> https://stackoverflow.com/questions/2676133/best-way-to-do-enum-in-sqlalchemy

Reply:
I'm now using enum for it in my new commit.